src/OrchardCore.Modules/OrchardCore.Forms/Views/Widget-Form.cshtml

```diff
@@ -0,0 +1,7 @@
+@{
+    Model.Classes.Add("mb-3");
```

Comment:
This is bootstrap, on the front-end, for a default template. Is that acceptable?

Or another option is to use "form-widget-input" as a class instead of "mb-3"

Reply:
And adding "widget-body" as a class would make it less a breaking change.

Reply:
as agreed. these files were removed from this PR

Reply:
I disagree to add bootstrap specific CSS class

Reply:
@hishamco please your comment in PR #12309 since the referenced code here is no longer part of this PR